client/disputes/utils.ts

```diff
@@ -76,7 +76,7 @@ export const isInquiry = ( dispute: Pick< Dispute, 'status' > ): boolean => {
  * Returns the dispute fee balance transaction for a dispute if it exists
  * and the deduction has not been reversed.
  */
-const getDisputeDeductedBalanceTransaction = (
+export const getDisputeDeductedBalanceTransaction = (
```

Comment:
Interesting … was this not accessible before?

Reply:
It was no longer used outside of this file once the "wrapper" function to get the formatted fee was introduced and used instead.

But now I have a use-case for it again.